ec2/ec2.go

```diff
@@ -560,6 +560,47 @@ func (ec2 *EC2) Instances(instIds []string, filter *Filter) (resp *InstancesResp
 	return
 }
 
+// The ModifyInstanceAttribute type encapsulates options for the respective
+// request in EC2.
+//
+// See http://goo.gl/cTxLa7 for more details.
+type ModifyInstanceAttribute struct {
+	InstanceId          string
+	SourceDestCheck     *bool
+	BlockDeviceMappings []InstanceBlockDeviceMapping
+}
+
+// ModifyInstanceAttribute modifies a single attribute of an instance in EC2.
+//
+// See http://goo.gl/cTxLa7 for more details.
+func (ec2 *EC2) ModifyInstanceAttribute(req *ModifyInstanceAttribute, filter *Filter) (resp *SimpleResp, err error) {
+	params := makeParams("ModifyInstanceAttribute")
+	params["InstanceId"] = req.InstanceId
+	if req.SourceDestCheck != nil {
+		params["SourceDestCheck.Value"] = fmt.Sprint(*req.SourceDestCheck)
+	}
+	if req.BlockDeviceMappings != nil {
```

Comment:
Yeah, I'm relying on the server to return an error. I don't think it's really worth adding code here to check as well.